vectordatasource/transform.py

```diff
@@ -1154,32 +1154,31 @@ def intracut(ctx):
 }
 
 
-# if the feature does not have a scale_rank attribute already,
+# if the feature does not have a min_zoom attribute already,
 # which would have come from a curated source, then calculate
 # a default one based on the kind of place it is.
-def calculate_default_place_scalerank(shape, properties, fid, zoom):
-    # don't override an existing attribute
-    scalerank = properties.get('scalerank')
-    if scalerank is not None:
+def calculate_default_place_min_zoom(shape, properties, fid, zoom):
+    min_zoom = properties.get('min_zoom')
+    if min_zoom is not None:
         return shape, properties, fid
 
     # base calculation off kind
     kind = properties.get('kind')
     if kind is None:
         return shape, properties, fid
 
-    scalerank = _default_scalerank_for_place_kind.get(kind)
-    if scalerank is None:
+    min_zoom = _default_min_zoom_for_place_kind.get(kind)
+    if min_zoom is None:
         return shape, properties, fid
 
-    # adjust scalerank for state / country capitals
+    # adjust min_zoom for state / country capitals
     if kind in ('city', 'town'):
         if properties.get('state_capital'):
```

Comment:
Since https://github.com/tilezen/vector-datasource/pull/1003/files was merged into master already, I'll update this here.